.github/workflows/pipeline.yml

```diff
@@ -22,42 +22,65 @@ jobs:
         flavor: ["prod", "dev"]
     steps:
     - name: Checkout compliantkubernetes-apps
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
+    - name: Set up Docker
+      uses: docker/setup-buildx-action@v2
+    - name: Build Docker image
+      uses: docker/build-push-action@v3
       with:
-        fetch-depth: 0 # Fetches all tags
+        cache-from: type=gha
+        context: ./pipeline
+        load: true
+        pull: false
+        push: false
+        tags: elastisys-ck8s-apps/app:latest
     - name: Init and configure config for compliantkubernetes-apps
-      uses: ./pipeline
-      with:
-        args: bash -c "./pipeline/sops-pgp-setup.bash && ./pipeline/init-unit-tests.bash"
+      run: /usr/bin/docker run --rm -e "CK8S_CONFIG_PATH" -e "CK8S_CLOUD_PROVIDER" -e "CI_CK8S_FLAVOR" -e "CK8S_PGP_FP" -e "GITHUB_RUN_ID" -e CI=true -v "/home/runner/work/compliantkubernetes-apps/compliantkubernetes-apps":"/github/workspace" -w /github/workspace elastisys-ck8s-apps/app:latest bash -c "./pipeline/sops-pgp-setup.bash && ./pipeline/init-unit-tests.bash"
       env:
         CK8S_CLOUD_PROVIDER: ${{ matrix.cloud }}
         CI_CK8S_FLAVOR: ${{ matrix.flavor }}
         CK8S_PGP_FP: 529D964DE0BBD900C4A395DA09986C297F8B7757
     - name: Run linters
-      uses: ./pipeline
-      with:
-        args: bash -c "./pipeline/sops-pgp-setup.bash && ./pipeline/lint.bash"
+      run: /usr/bin/docker run --rm -e "CK8S_CONFIG_PATH" -e "CK8S_CLOUD_PROVIDER" -e "CI_CK8S_FLAVOR" -e "CK8S_PGP_FP" -e "GITHUB_RUN_ID" -e CI=true -v "/home/runner/work/compliantkubernetes-apps/compliantkubernetes-apps":"/github/workspace" -w /github/workspace elastisys-ck8s-apps/app:latest bash -c "./pipeline/sops-pgp-setup.bash && ./pipeline/lint.bash"
```

Comment:
Those there exist a "uses" that takes a docker image?
My thought is just to clear up the one liner :smile:

Reply:
Nvm did not really find any good option

Reply:
Yeah, that is the problem with this approach.